deps.bzl

```diff
@@ -7,16 +7,17 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
 _LABELS = {
-    "darwin_amd64": "darwin.x86_64",
-    "linux_amd64": "linux.x86_64",
-    "linux_arm64": "linux.aarch64",
-    # darwin_arm64 shellcheck binaries are not distributed from GitHub releases.
-    # windows_amd64 shellcheck binaries are not distributed from GitHub releases.
+    "darwin_aarch64": "darwin.aarch64",
+    "darwin_amd64":   "darwin.x86_64",
```

Comment:
I did notice that this code-base uses `amd64` and `x86_64` interchangably. I've tried to preserve the existing style, but I would personally recommend switching to just x86_64 to keep stuff simple.

Reply:
Using x86_64 everywhere SGTM.

Reply:
working on that now